sphinx/source/docs/user_guide/jupyter.rst

```diff
@@ -129,24 +129,24 @@ to the URL of your JupyterHub installation. By default, JupyterHub will set
         full_url = urllib.parse.urljoin(user_url, proxy_url_path)
         return full_url
 
-Finally, you can pass the function you defined in step 2 to |show|
-as the notebook_url keyword argument, which Bokeh will call while
-setting up the server and creating the URL for loading the graph:
+3. Pass the function you defined in step 2 to the |show| function
+   as the notebook_url keyword argument, which Bokeh will call while
+   setting up the server and creating the URL to load a graph:
 
-.. code-block:: python
+   .. code-block:: python
 
     show(obj, notebook_url=remote_jupyter_proxy_url)
 
-At this point, the Bokeh graph should load and execute python
+Now the Bokeh graph should load and execute python
 callbacks defined in your JupyterLab environment.
 
 Trusting Notebooks
 ~~~~~~~~~~~~~~~~~~
 
-Depending on the version of the notebook in use, it may be necessary to
-"trust" the notebook in order for Bokeh plots to re-render when the
-notebook is closed and subsequently re-opened. The "Trust Notebook" option
-is typically located under the "File" menu:
+Depending on the version of the notebook you use, you may have to
+"trust" the notebook for Bokeh plots to re-render when the
```

Comment:
the quotes here seem a bit like scare quotes. If you have the bandwidth for a small change it might be nice for the text "trust the notebook" to be a link to some Jupyter docs about trusting notebooks (with no quotes)

Reply:
Good point. I'll look into this.